doc/source/conf.py

```diff
@@ -16,16 +16,15 @@
 from recommonmark.transform import AutoStructify
 
 sys.path.insert(0, os.path.abspath(".."))
-import qibo
 
 # -- Project information -----------------------------------------------------
 
-project = "qibo"
+project = " "
```

Comment:
Please follow the approach in https://github.com/qiboteam/qibo/pull/745, or simply remove those changes in this PR.

Reply:
I suggest to remove them, since I am making these changes in dedicated PR.